src/test/java/com/faforever/client/test/AbstractPlainJavaFxTest.java

```diff
@@ -106,6 +107,37 @@ protected void loadFxml(String fileName, Callback<Class<?>, Object> controllerFa
     }
   }
 
+  protected void loadFxml(String fileName, Callback<Class<?>, Object> controllerFactory, VaultEntityController<?> controller) throws IOException {
```

Comment:
Could we not unify this with the other loadFXMl there is not realy too much of a difference

Reply:
combined